docker/docker-mssql-2017.dockerfile

```
@@ -1,5 +1,8 @@
 FROM mcr.microsoft.com/mssql/server:2017-latest
 
-COPY --chmod=440 certs/server.* /certs/
-COPY --chmod=440 certs/customCA.* /certs/
+USER root
```

Hadolint (2.14.0)

[warning] Last USER should not be root

(DL3002)